adapters/repos/db/inverted/tracker/propertyIdTracker.go

```diff
@@ -0,0 +1,149 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package tracker
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"sync"
+)
+
+type JsonPropertyIdTracker struct {
+	path        string
+	LastId      uint64
+	PropertyIds map[string]uint64
+	sync.Mutex
+}
+
+func NewJsonPropertyIdTracker(path string) (*JsonPropertyIdTracker, error) {
+	t := &JsonPropertyIdTracker{
+		path:        path,
+		PropertyIds: make(map[string]uint64),
+		LastId:      1,
+	}
+
+	log.Printf("Loading property id tracker from %v\n", path)
```

Comment:
There should be some logger lower down the stack which should be propagated up, and added, to this type. This will make log collection much easier in the future